### ISSUE TITLE
add secondary prompt for delete

### ISSUE THREAD

**Issue by satra** [dandi/dandiarchive#910]
at present clicking delete prompts a display with a yes/cancel option. given the size of files and upcoming actual delete from blob store, can we add another layer of protection.

type delete in a text box to actually delete. this is similar to many other sites.

**Comment by yarikoptic** [comment]
hm, not certain it is worth that much protection: users can "delete" via API calls, `dandi delete`, `dandi upload` (although there at least there is an explicit option to be given for overwrites). If user is to remove a good number of files in web ui, it would become an impediment, unless we also add some checkbox "assume it is ok for all other files to be deleted in this session" or alike (and store that in a user cookie)

**Comment by satra** [comment]
that is true of s3 /githubas well (i.e. with the API you can hit delete anytime), but if you use a browser interface all of these services (aws console, github ui) have a second layer of protection.

i think it's just good practice that has emerged.

**Comment by waxlamp** [comment]
I agree this would be a good safeguard (even if it does not apply to direct API calls).

In other projects I have done this by creating a random phrase for the user to type in per delete action, whether that is a direct deletion of a single item, or a bulk delete (using a checkbox system).
